tools/sigstore-policy-tester/README.md

```diff
@@ -0,0 +1,44 @@
+# sigstore-policy-tester extension
+
+## Installation
+
+See [Installing Extensions](https://github.com/siderolabs/extensions#installing-extensions).
+
+## Usage
+
+```yaml
+machine:
+  files:
+    - content: |
+        [plugins]
+          [plugins."io.containerd.image-verifier.v1.bindir"]
```

Comment:
I also would like to check how this plugs in into the image pull process, there are multiple paths in the CRI/Talos itself before we declare this as a production solution.

Reply:
From the docs (https://github.com/containerd/containerd/blob/main/docs/image-verification.md#image-pull-judgement), it won't pull an image unless it passes the checks in the bindir

> Return an exit code of 0 to allow the image to be pulled and any other exit code to block the image from being pulled.

Reply:
The problem is that containerd doesn't pull images ;) 

In fact, they have Transfer API which is only used by `ctr`, and direct pull, when the actual pull process happens outside of containerd. This API is being used by CRI & Talos. So image verification might be pulled into CRI, but it I doubt it covers all paths.

Reply:
According to https://youtu.be/fDDSg2NVwO4?t=322, containerd 2.1 will resolve this. 

In any case, the functionality of image verification is what I'd like and it doesn't have to be this way. That said, there's work along this path to achieve this functionality